### PR TITLE
doc: update some api format & description

### LINKED ISSUE #51: gremlin api 文档 有误
[gremlin api 文档](https://hugegraph.github.io/hugegraph-doc/clients/restful-api/gremlin.html)中异步执行的 Response Status 应该是201
原文如下：

> 7.1.3 向HugeGraphServer发送gremlin语句（POST），异步执行
Method & Url
POST http://localhost:8080/graphs/hugegraph/jobs/gremlin
查询顶点
>
>Request Body
>{
    "gremlin": "g.V('1:marko')",
    "bindings": {},
    "language": "gremlin-groovy",
    "aliases": {}
}
>注意：
>
> >异步执行Gremlin语句暂不支持aliases，可以使用 graph 代表要操作的图，也可以直接使用图的名字， 例如 hugegraph; 另外g代表 traversal，等价于 graph.traversal() 或者 hugegraph.traversal()
>
> Response Status
> 200

### PR DESCRIPTION
**Pull request by imbajin**
Fixed #51 #52 and some desc:
1. remove some outdated design conception
2. adjust `REST-API` &  Guide` position
3. add some default params for client
4. add some comments for `skip_degree`
5. add comment for Cassandra page
6. add auth-api request example & link page in rest-menu

### CODE REVIEW

File: config/config-authentication.md [hunk 1]
```diff
@@ -7,15 +7,20 @@ HugeGraph支持动态创建用户、用户组、资源，支持动态分配或
...
 ### 配置用户认证
 
 HugeGraph默认不启用用户认证功能，可通过修改配置文件来启用该功能。内置实现了`StandardAuthenticator`和`ConfigAuthenticator`两种模式，`StandardAuthenticator`模式支持多用户认证与细粒度权限控制，`ConfigAuthenticator`模式支持简单的用户权限认证。此外，开发者可以自定义实现`HugeAuthenticator`接口来对接自身的权限系统。
 
-用户认证方式均采用 [HTTP Basic Authentication](https://zh.wikipedia.org/wiki/HTTP%E5%9F%BA%E6%9C%AC%E8%AE%A4%E8%AF%81)。
+用户认证方式均采用 [HTTP Basic Authentication](https://zh.wikipedia.org/wiki/HTTP%E5%9F%BA%E6%9C%AC%E8%AE%A4%E8%AF%81) ，简单说就是在发送 HTTP 请求时在 `Authentication` 设置选择 `Basic` 然后输入对应的用户名和密码，对应 HTTP 明文如下所示 :
+
+```http
+GET http://localhost:8080/graphs/hugegraph/schema/vertexlabels
+Authorization: Basic admin 123456
```
Comment (zhoney) [inline config/config-authentication.md:22]: use xxxxxx instead of 123456

File: clients/restful-api/traverser.md [hunk 2]
```diff
@@ -1433,7 +1433,7 @@ GET http://localhost:8080/graphs/{graph}/traversers/paths?source="1:marko"&targe
 	- labels：边的类型列表
 	- properties：通过属性的值过滤边
 	- max_degree：查询过程中，单个顶点遍历的最大邻接边数目，默认为 10000 (注: 0.12版之前 step 内仅支持 degree 作为参数名, 0.12开始统一使用 max_degree, 并向下兼容 degree 写法)
-	- skip_degree：查询过程中需要跳过的顶点的最小的边数目，即当顶点的边数目大于 skip_degree 时，跳过该顶点，可用于规避超级点，选填项，默认为0，表示不跳过任何点
+	- skip_degree：用于设置查询过程中舍弃超级顶点的最小边数，即当某个顶点的邻接边数目大于 skip_degree 时，完全舍弃该顶点，选填项，默认为0 (关闭)，表示不跳过任何点 (注意:  开启此配置后，会无视 max_degree 参数优先确认当前点是否需要舍弃，并产生实际遍历开销，对查询性能影响可能有较大影响，请确认理解后再开启)
```
Comment (zhoney) [inline clients/restful-api/traverser.md:1436]: 不要用“无视”。

开启此配置后，遍历时会尝试访问一个顶点的skip_degree条边，而不仅仅是max_degree条边，对查询性能.....

**Review by zhoney — CHANGES_REQUESTED** [review]

**Review by zhoney — DISMISSED** [review]

File: clients/hugegraph-api.md [hunk 3]
```diff
@@ -3,12 +3,12 @@
...
 图数据的增删改查，遍历算法，变量，图操作及其他操作。
 
-- Schema
+- Graph Schema
   - [PropertyKey](restful-api/propertykey.md)
   - [VertexLabel](restful-api/vertexlabel.md)
   - [EdgeLabel](restful-api/edgelabel.md)
   - [IndexLabel](restful-api/indexlabel.md)
-- Graph
+- Graph Data
```
Comment (javeme) [inline clients/hugegraph-api.md:11]: the "Data" word is too general, prefer to keep the origin title.

File: clients/hugegraph-client.md [hunk 4]
```diff
@@ -13,12 +13,15 @@ HugeGraph-Client 是操作 graph 的总入口，用户必须先创建出 HugeGra
 ```java
 // HugeGraphServer地址："http://localhost:8080"
 // 图的名称："hugegraph"
-HugeClient hugeClient = new HugeClient("http://localhost:8080", "hugegraph");
+HugeClient hugeClient = HugeClient.builder("http://localhost:8080", "hugegraph")
+                                  .configTimeout(20) // 默认 20s
```
Comment (javeme) [inline clients/hugegraph-client.md:17]: 超时默认 20s

File: clients/hugegraph-client.md [hunk 5]
```diff
@@ -13,12 +13,15 @@ HugeGraph-Client 是操作 graph 的总入口，用户必须先创建出 HugeGra
...
+                                  .configUser("", "") // 默认未开启用户权限
```
Comment (javeme) [inline clients/hugegraph-client.md:18]: "**", "**"

File: clients/restful-api/edge.md [hunk 6]
```diff
@@ -541,7 +541,7 @@ GET http://127.0.0.1:8080/graphs/hugegraph/graph/edges?page=002500100753313a6a6f
 }
 ```
 
-此时`"page": null`表示已经没有下一页了。
+此时`"page": null`表示已经没有下一页了 (注: 后端为 Cassandra 时，为了优化性能，直接访问最后一条数据时 page 仍不会显示为 null，其他情况一致)
```
Comment (javeme) [inline clients/restful-api/edge.md:544]: 如果后端为 Cassandra ，当返回正好为最后一页时，page 可能并不为空，通过该 page 再次请求数据时会返回【空数据】及 【page=null】。本行为主要是基于性能考虑。

File: clients/restful-api/traverser.md [hunk 7]
```diff
@@ -252,7 +252,7 @@ GET http://localhost:8080/graphs/{graph}/traversers/kout?source="1:marko"&max_de
 	- labels：边的类型列表
 	- properties：通过属性的值过滤边
 	- max_degree：查询过程中，单个顶点遍历的最大邻接边数目，默认为 10000 (注: 0.12版之前 step 内仅支持 degree 作为参数名, 0.12开始统一使用 max_degree, 并向下兼容 degree 写法)
-	- skip_degree：查询过程中需要跳过的顶点的最小的边数目，即当顶点的边数目大于 skip_degree 时，跳过该顶点，可用于规避超级点，选填项，默认为0，表示不跳过任何点
+	- skip_degree：用于设置查询过程中舍弃超级顶点的最小边数，即当某个顶点的邻接边数目大于 skip_degree 时，完全舍弃该顶点，选填项，默认为0 (关闭)，表示不跳过任何点 (注意:  开启此配置后，遍历时会尝试访问一个顶点的 skip_degree 条边，而不仅仅是 max_degree 条边，这样有额外的遍历开销，对查询性能影响可能有较大影响，请确认理解后再开启)
```
Comment (javeme) [inline clients/restful-api/traverser.md:255]: "(关闭)" => "(不启用)"

Comment (javeme) [inline clients/restful-api/traverser.md:255]: "选填项，" 后加上： ”如果开启时，要求满足 skip_degree >= max_degree 约束“

File: clients/restful-api/traverser.md [hunk 8]
```diff
@@ -454,7 +454,7 @@ GET http://localhost:8080/graphs/{graph}/traversers/kneighbor?source=“1:marko
...
```
Comment (javeme) [inline clients/restful-api/traverser.md:457]: 类似

File: clients/restful-api/vertex.md [hunk 9]
```diff
@@ -604,7 +604,7 @@ GET http://localhost:8080/graphs/hugegraph/graph/vertices?page=001000100853313a7
 }
 ```
 
-此时`"page": null`表示已经没有下一页了。
+此时`"page": null`表示已经没有下一页了 (注: 后端为 Cassandra 时，为了优化性能，直接访问最后一条数据时 page 仍不会显示为 null，其他情况一致)
```
Comment (javeme) [inline clients/restful-api/vertex.md:607]: 类似

File: clients/hugegraph-api.md (hunk 3 above)
Comment (imbajin) [reply clients/hugegraph-api.md:11]: seems `Graph` is also general?  maybe consider `Graph Elements` / `Graph Vertex & Edge` / `Graph Source Data` ?

File: clients/hugegraph-client.md (hunk 5 above)
Comment (imbajin) [reply clients/hugegraph-client.md:18]: seems there is no difference?

**Review by javeme — APPROVED** [review]

**Review by Linary — APPROVED** [review]

**Pull request merged**